src/schema.jl

```diff
@@ -53,6 +53,19 @@ Base.merge!(a::Schema, b::Schema) = (merge!(a.schema, b.schema); a)
 Base.keys(schema::Schema) = keys(schema.schema)
 Base.haskey(schema::Schema, key) = haskey(schema.schema, key)
 
+function Base.:(==)(first::Schema, second::Schema)
+    first === second && return true
+    first.schema === second.schema && return true
+    length(first.schema) != length(second.schema) && return false
+    for key in keys(first)
+        !haskey(second, key) && return false
+        second[key] != first[key] && return false
+    end
+    true
```

Comment:
Isn't this identical to `first == second`? If not, maybe worth a comment. (Note that this throws if the dict contains `missing`, maybe that's OK though.)

Reply:
Hmmm I think you're right.  Probably fine to just check `first.schema == second.schema` since that's really what this check is about